src/isIsogram.test.js

```diff
@@ -6,4 +6,58 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for the isogram', () => {
+    const result = isIsogram('playgrounds');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return false for the non isogram', () => {
+    const result = isIsogram('look');
+
+    expect(result).toBe(false);
+  });
+
+  it('should return ture for the empty strng', () => {
```

Comment:
There is a typo in the test description: 'ture' should be 'true', and 'strng' should be 'string'.